ejabberd_bridge/management/commands/ejabberd_auth.py

```diff
@@ -47,33 +49,37 @@ def to_ejabberd(self, answer=False):
         sys.stdout.flush()
 
     def auth(self, username=None, server="localhost", password=None):
-        self.logger.debug("Authenticating %s with password %s on server %s" % (username, password, server))
-        #TODO: would be nice if this could take server into account
-        user = authenticate(username=username, password=password)
-        return user and user.is_active
+        self.logger.debug("Authenticating %s on server %s" % (username, server))
+
+        try:
+            user_obj = get_user_model().objects.get(id=username)            
```

Comment:
it would be nice if you can call `get_user_model` only once as global to the whole file